packages/create-playwright/src/index.ts

```diff
@@ -182,14 +182,14 @@ Happy hacking! 🎭`);
   }
 }
 
-export function commandToRunTests(packageManager: 'npm' | 'yarn') {
+function commandToRunTests(packageManager: 'npm' | 'yarn') {
   if (packageManager === 'yarn')
     return `yarn ${PACKAGE_JSON_TEST_SCRIPT_CMD}`;
   return `npm run ${PACKAGE_JSON_TEST_SCRIPT_CMD}`;
 }
 
 (async () => {
-  const rootDir = determineRootDir();
+  const rootDir = path.resolve(process.cwd(), process.argv[2]);
```

Comment:
```suggestion
  const rootDir = process.argv[2] ? path.resolve(process.cwd(), process.argv[2]) : process.cwd();
```

Reply:
Failing on the CI because of

```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received function wrappedCwd
    at validateString (internal/validators.js:124:11)
    at Object.resolve (path.js:1074:7)
    at REPL13:1:6
    at Script.runInThisContext (vm.js:134:12)
    at REPLServer.defaultEval (repl.js:486:29)
    at bound (domain.js:416:15)
    at REPLServer.runBound [as eval] (domain.js:427:12)
    at REPLServer.onLine (repl.js:819:10)
    at REPLServer.emit (events.js:387:35)
    at REPLServer.emit (domain.js:470:12) {
  c
```